arangod/Cluster/SynchronizeShard.cpp

```diff
@@ -566,7 +566,8 @@ static arangodb::ResultT<SyncerId> replicationSynchronize(
   SyncerId syncerId{syncer->syncerId()};
 
   try {
-    Result r = syncer->run(configuration._incremental);
+    std::string const context = "synchronization of shard " + col->name() + " in database " + database;
+    Result r = syncer->run(configuration._incremental, context.c_str());
```

Comment:
do we really want to downmix to a c-string here?

Reply:
Technically it is not necessary, and passing an empty std::string would have also worked.
I was using a pointer here intentionally to convey the idea that the value can be unset (i.e. nullptr).